docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -83,20 +84,47 @@ code: |
     process_custom_people(people_variables.true_values(), fields, built_in_fields_used)
   process_people_variables = True    
 ---
-id: opening screen
+id: weaver_intro
+field: weaver_intro
 question: |
- Interview Weaver: Document Assembly Line
+  Welcome to the Interview Weaver 
 subquestion: |
-  Use this tool to turn a PDF or Word document into a draft
-  interactive online form.
+  The Interview Weaver is part of the [Doc Assembly Line](https://suffolklitlab.org/doc-assembly-line/) project. This powerful tool quickly turns a PDF or Word document into an interactive online form, which is supported at runtime by other _**Doc Assembly Line**_ tools.
   
-  This tool is part of the [Doc Assembly 
-  Line](https://suffolklitlab.org/doc-assembly-line/)
-  project.
+  **What the Weaver does:**  
   
-  **If your file is PDF, we'll validate your PDF fields first.**
-fields:
-  - Select a PDF or Word template: template_upload
+  1. It turns your form into a draft DocAssembly interview. The end product of the Weaver is a Docassemble package for you to download.
+  2. If your file is PDF, the Weaver will validate your PDF field labels first.
+  3. The draft interview will include all the dependencies it needs, which are additional Docassemble packages in the _**Doc Assembly Line**_ project that offer convenient and useful functionality and visual effects for your interview.
+  4. As such, variable names in the generated draft interview are standardized.
+  
+  **What you need to do:**
+  <ol>
+    <li>Upload a prepared document and answer all the questions on the screens.</li>
+    <li>At the end of the Weaver interview, download the package then upload it to your playground. It should run out of box.</li>
+    <li>In many cases you may need to customize the generated YAML file. Test each screen to determine what needs to be changed.</li>
+    <li>You can safely:
+      <ol type="a">        
+        <li>reword titles/text in the question blocks and the attachment docx files. You can simply copy the text from the screens or docx files, then search it in the YAML file or docx files to make changes.</li>   
+        <li>reorder questions, remove/add variables on the Review screen, etc.</li>
+        <li>change the "interview order code block" block.</li>
+      </ol></li>
+    <li>Generally you want to avoid:
+      <ol type="a">
+        <li>changing code in the "main order" block.</li>
+        <li>changing variable names that were generated by the Weaver. However, you can add new variables then use your own code to handle them, and remember to use "depends on" if the new variable is associated with any variable on the Review screen.
+        <li>changing the "include" block unless you know what you are doing.
+        <li>deleting generated code until you are certain. Comment it out first and test to ensure it doesn't break the interview.</li>    
+      </ol></li>
+  </ol>
```

Comment:
Markdown can do ordered lists, right? You just need to get the indentation right, which is tricky. But it's much easier to edit than raw HTML.

Reply:
Right, it does <ol> but seems to be only on the first level, not the nested ones. If you know the syntax, that would be great.

Reply:
I got it working: 
```
question: |
  What is your name?
subquestion: |
  1. Testing this feature.
      1. And a little more
      2. And some more
  2. Viola
```
You do need 4 spaces as indention, 2 won't work.